CHANGELOG.md

```diff
@@ -79,7 +82,7 @@
         * Add error `.canceled` = 5
         * Add error `.invalidAPIClient` = 6
         * Add error `.cannotCastRequest`= 7
-        * Add error `.jsonSerializationFailure` = 8
+        * Add error `.jsonSerializationFailure` = 8    
```

Comment:
We can probably revert this line

Reply:
fixed in commit ba406a4e001c6139be262b107668a8048acce0c2